### ISSUE TITLE
--model_directory just got broken

### ISSUE THREAD

**Issue by anneriekeschipperen893** [jordimas/whisper-ctranslate2#6]
Hello, i just updated to latest version and --model_directory argument seems to be broken. 
```
error: unrecognized arguments: --model_directory
```

**Comment by jordimas** [comment]
Hello,

It works for me. I can specify a model directory:

_whisper-ctranslate2 inaguracio2011.mp3 --model_directory /home/jordi/sc/transcribe/whisper-ctranslate2/models/sc-medium-ca-200steps/_

and if I do not provide a directory it:

_whisper-ctranslate2 inaguracio2011.mp3 --model_directory_

it says:

_whisper-ctranslate2: error: argument --model_directory: expected one argument_

What's exactly the command line that you are trying? Thanks

**Comment by nfranke** [comment]
If I install via either installation method (plain pip or pip via github) I also get:

```
Traceback (most recent call last):
  File "/home/me/.local/bin/whisper-ctranslate2", line 5, in <module>
    from src.whisper_ctranslate2.whisper_ctranslate2 import main
  File "/home/me/.local/lib/python3.8/site-packages/src/whisper_ctranslate2/whisper_ctranslate2.py", line 4, in <module>
    from .models import Models
  File "/home/me/.local/lib/python3.8/site-packages/src/whisper_ctranslate2/models.py", line 10, in <module>
    class Models:
  File "/home/me/.local/lib/python3.8/site-packages/src/whisper_ctranslate2/models.py", line 35, in Models
    def get_list(self) -> list[str]:
TypeError: 'type' object is not subscriptable
```

**Comment by jordimas** [comment]
Thanks for the details.

I have just released version 0.1.4. If you can please confirm if fixes the issue for you. Thanks

**Comment by anneriekeschipperen893** [comment]
> Thanks for the details.
> 
> I have just released version 0.1.4. If you can please confirm if fixes the issue for you. Thanks

Yes, works good now after uninstall and install again. Upgrade alone did not yield any results.
--model_directory works as well

Thanks

**Comment by jordimas** [comment]
Thanks for reporting and test it!
